documentation/plain_rules.md

```diff
@@ -190,6 +191,197 @@ First time an update action using trust token is triggered, Perseo interacts wit
 
 It could happen (in theory) that a just got auth token also produce a 401 Not authorized, however this would be an abnormal situation: Perseo logs the problem with the update but doesn't try to get a new one from Keystone. Next time Perseo triggers the action, the process may repeat, i.e. first update attemp fails with 401, Perseo requests a fresh auth token to Keystone, the second update attemp fails with 401, Perseo logs the problem and doesn't retry again.
 
+NGSIv2 example:
+```json
+"action":{
+        "type":"update",
+        "parameters":{
+            "id":"${id}_mirror",
+            "version": 2,
+            "attributes": [
+                {
+                    "name":"abnormal",
+                    "type":"number",
```

Comment:
Maybe this examples should change to "Number" (with capital letter)? Or the type processing logic is case insensitive?

Reply:
Looking to switch() at https://github.com/telefonicaid/perseo-fe/pull/305/files#diff-43c1a406c68dbf35ee797980d6caf5deR267 it seems case sensitive.

Reply:
Yes. is case sensitive. This "number" in the type is an error.

Fixed in 3e85263